scripts/core/activity/activity.js

```diff
@@ -172,6 +173,18 @@ function SuperdeskProvider($routeProvider, _) {
                 return privileges.userHasPrivileges(activity.privileges);
             }
 
+            function checkActivityEnabled(activity) {
+                if (!_.get(config, 'activity')) {
+                    return true;
+                }
+
+                if (_.isUndefined(config.activity[activity._id])) {
```

Comment:
I think I would rather return `false` here because that will happen with new fields..
so either you have them all or you hand pick and then imo there should be no surprises..